charts/hail-search/values.yaml

```diff
@@ -68,6 +78,14 @@ affinity: |-
 nodeSelector: 
   {}
 
+sourceCachedReferenceDatasetQueriesDir: 'gs://seqr-reference-data/v03'
+cachedReferenceDatasetQueries:
+  GRCh38/SNV_INDEL:
+    - 'GRCh38/cached_reference_dataset_queries/clinvar_path_variants.ht'
+  GRCh38/MITO:
```

Comment:
I've just realized that we're not splitting our `clinvar` into `clinvar_mito` and a normal `clinvar` for `SNV_INDEL`.  We absolutely can:

```
In [14]: ht = hl.read_table('gs://seqr-datasets/v03/GRCh38/SNV_INDEL/runs/manual__2023-10-05T16-59-58.006799+00-00/clinvar_path_variants.ht')
In [15]: ht.group_by(contig=ht.locus.contig).aggregate(n=hl.agg.count()).show()
2023-10-12 12:57:50.598 Hail: INFO: Coerced sorted dataset
+---------+-------+
| contig  |     n |
+---------+-------+
| str     | int64 |
+---------+-------+
| "chr1"  | 16421 |
| "chr10" |  6284 |
...
| "chr5"  | 10149 |
| "chr6"  |  9322 |
| "chrM"  |   271 |
| "chrX"  | 17140 |
| "chrY"  |    38 |
+---------+-------+
```